fms/modules/linear.py

```diff
@@ -9,6 +9,7 @@
 
 __type_factory_map: dict[str, Callable] = {}
 __type_sharding_map: dict[str, Callable] = {}
+__type_rope_params_map: dict[str, list[str]] = {}
```

Comment:
Will this be re-used other than for llama?

Reply:
no, will probably remove and specialize